File: guide-projects/plus-repository-guide/src/main/java/com/navercorp/spring/data/jdbc/plus/repository/guide/Application.java
package com.navercorp.spring.data.jdbc.plus.repository.guide;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Primary;
import org.springframework.data.relational.core.dialect.Dialect;
import org.springframework.data.relational.core.dialect.MySqlDialect;

@SpringBootApplication
public class Application {
	public static void main(String[] args) {
		SpringApplication.run(Application.class, args);
	}

	@Bean
	@Primary
	public Dialect mysqlDialect() {
		return MySqlDialect.INSTANCE;
	}
}

File: guide-projects/plus-sql-kotlin-guide/src/main/kotlin/com/navercorp/spring/data/jdbc/plus/sql/guide/Application.kt
package com.navercorp.spring.data.jdbc.plus.sql.guide

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Primary
import org.springframework.data.relational.core.dialect.Dialect
import org.springframework.data.relational.core.dialect.MySqlDialect

@SpringBootApplication
class Application {

    fun main(args: Array<String>) {
        runApplication<Application>(*args)
    }

    @Bean
    @Primary
    fun mysqlDialect(): Dialect {
        return MySqlDialect.INSTANCE
    }
}
